CHANGELOG.md

```diff
@@ -0,0 +1,22 @@
+# Changelog
+This doc describes all notable changes to Diem Core and Diem Framework between releases.
+
+
+## [diem-core-v1.1.5] - 2021-03-02
+*[Added]*
+
+* Add NetworkIdentity to metrics dashboard (#7669)
+* Add health check of node status via json-rpc (#7498)
+* Add genesis smoke test to backup & restore tooling (#7465)
+
+*[Changed]*
+
```

Comment:
We should probably add "Increase max number of child accounts to 2^16 form 2^8"

Reply:
that change is not in this release, see the diff view https://github.com/diem/diem/compare/diem-core-v1.1.4...diem-core-v1.1.5

the child account size change went in pretty early, before we spin up Mainnet